eng/common/Update-Change-Log.ps1

```diff
@@ -95,7 +100,7 @@ function Get-NewChangeLog( [System.Collections.ArrayList]$ChangelogLines, $Versi
       exit(0)
    }
 
-   if (($ReplaceVersion -eq $True) -and ($Unreleased -eq $False) -and $CurrentTitle.Contains($version) -and (-not $CurrentTitle.Contains($UNRELEASED_TAG))) {
+   if (($ReplaceVersion -eq $True) -and ($Unreleased -eq $False) -and $CurrentTitle.Contains($version) -and (-not $CurrentTitle.Contains($UNRELEASED_TAG)) -and (-not $ReleaseDate)) {
```

Comment:
I must say I'm have a hard time understanding this condition :)

Why did you need to add the ReleaseTag option to it? Is it because you are trying to change the release date?

Reply:
This condition prevents the changelog line from being updated when the only difference is the date. I want to be able to change the date of the existing entry that is not `unreleased`.